Functions/fetchData.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:io';
+import 'dart:convert';
+
+Map<int,Map<String,dynamic>> readingFile() {
+  
+  Map<int,Map<String,dynamic>> data = {};
+
+  print(".............Geting old record.......");
+ for(int i=0; i<2;i++){
+   print(".");
+   sleep(Duration(seconds:1));
+ }
+
+  try {
+  var path = 'dataBaseUser.txt';
```

Comment:
use static const for filenames, and other constants which wont change ever. 
I would suggest you to Read more on `final` and `const` keyword in dart, in order to use them in best way possible.